### ISSUE TITLE
Searching for square ID, which appears only in the DB?

### ISSUE THREAD

**Issue by pongraczi** [tkrebs/ep3-bs#170]
Hi,  
I realized, when one wants to filter bookings for only one square, the only filter option is the square_id.  
Unfortunately, this information (sid) is not visible on the front/middle/backend, only in the database directly.  
Is that possible to include other search options, like:  
- priority
- name ?
Thanks.

**Comment by tkrebs** [comment]
Indeed, the ID is not easy to determine. In most cases, it is sequental (1,2,3, ...), but not necessarily. This should be made more straightforward. I have added it as a enhancement.

**Comment by pongraczi** [comment]
Consider the following additional filter possibilities:  
- searching for square priority (already visible and user can manipulate)
- searching for square name (free text, so, anything could be inside)

Additional possibility:  
- show the square ID on the square list and when edit a square details (a read-only label could be ok)
